pkgs/misc/tmux-plugins/default.nix

```diff
@@ -377,6 +377,40 @@ in rec {
     };
   };
 
+  tmux-fzf = mkDerivation {
+    pluginName = "tmux-fzf";
+    version = "unstable-2020-11-12";
+    src = fetchFromGitHub {
+      owner = "sainnhe";
+      repo = "tmux-fzf";
+      rev = "570f8baff3cb176d7339eb42cddc98e5695458f7";
+      sha256 = "0vavbmwhpqxk26avzbnjjswsv6cd7s0sdilwppxqjdif2zm6jg9k";
+    };
+    postInstall = ''
+      find $target -type f -print0 | xargs -0 sed -i -e 's|fzf |${pkgs.fzf}/bin/fzf |g'
+      find $target -type f -print0 | xargs -0 sed -i -e 's|sed |${pkgs.gnused}/bin/sed |g'
+    '';
+     meta = {
+      homepage = "https://github.com/sainnhe/tmux-fzf";
+      description = "Use fzf to manage your tmux work environment! ";
+      longDescription =
+        ''
+        Features:
+        * Manage sessions (attach, detach*, rename, kill*).
+        * Manage windows (switch, link, move, swap, rename, kill*).
+        * Manage panes (switch, break, join*, swap, layout, kill*, resize).
+        * Multiple selection (support for actions marked by *).
+        * Search commands and append to command prompt.
+        * Search key bindings and execute.
+        * User menu.
+        * Popup window support.
+      '';
+      license = stdenv.lib.licenses.mit;
+      platforms = stdenv.lib.platforms.unix;
+      maintainers = with stdenv.lib.maintainers; [ kyleondy ];
```

Comment:
You need to add yourself to the maintainers list.

Reply:
Which you can find [here](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) and should be updated in its own commit with the message `maintainers: add kyleondy`.

You can find your `githubId` with `curl -s https://api.github.com/users/kyleondy | jq '.id'`